test/Interop/C/modules/Inputs/SubModule.swift

```diff
@@ -0,0 +1,2 @@
+import ForeignA
```

Comment:
Could you rename it to something other than "SubModule"? "Submodule" has a particular meaning, but here it is not that.

Reply:
Donel